docs/operator-manual/rbac.md

```diff
@@ -30,7 +30,28 @@ Breaking down the permissions definition differs slightly between applications a
 
 Resources: `clusters`, `projects`, `applications`, `repositories`, `certificates`, `accounts`, `gpgkeys`
 
-Actions: `get`, `create`, `update`, `delete`, `sync`, `override`, `action`
+Actions: `get`, `create`, `update`, `delete`, `sync`, `override`,
+`action/<path/to/action>`
+
+#### Application resources
+
+The resource path for application objects is of the form
+`<project-name>/<application-name>`.
+
+Delete access to sub-resources of a project, such as a rollout or a pod, cannot
+be managed granularly. `<project-name>/<application-name>` grants access to all
+subresources of a project.
```

Comment:
I have no evidence that this is true. Please correct me and I will document it.

Reply:
I now disagree with myself. I think this is the only change needed.

```suggestion
subresources of an application.
```